1002-badge-design-guidance.md

```diff
@@ -0,0 +1,24 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-15
+updated: 2024-10-21
+---
+
+# Badge Design Guidance
+
+[Badge](https://clarity.design/documentation/badge) component is a dynamic status modifier to other components that indicate the number of instances of that component.
+
+## Guidance
+
+- **Must** use badge to highlight the count, while a [label](https://clarity.design/documentation/label) can be used to display metadata, which is usually text content. A label can contain a badge.
+- **Must** pair badge with a [label](https://clarity.design/documentation/label) or text that conveys the information.
+- **Must** use 99+ instead of the number, if number is over 99.
```

Comment:
```suggestion
- Design teams **must** use "99+" instead of displaying the actual number when the count exceeds 99. This keeps the interface clean and avoids overwhelming users with large numbers.
```